src/modules/bar/barNPCs.ts

```diff
@@ -1685,7 +1685,7 @@ export let GrumpyElfDialog: Dialog[] = [
   },
   {
     name: 'no-xplaza',
-    text: "Alright then, all good! If you change your mind, head to Xmas Plaza ("+CoordsEnum.XmasPlazaCoords+") and talk to the elf in charge.",
+    text: "Alright then, all good! If you change your mind, head to Xmas Plaza located at ("+CoordsEnum.XmasPlazaCoords+") and talk to the elf in charge.",
```

Comment:
was line breaking on the "negative sign" even though it was "-20,80" with no spaces.  had to add text to push it to not break in middle